deploy/manifests/base/kong-ingress-dbless.yaml

```diff
@@ -54,12 +62,12 @@ spec:
           containerPort: 8443
           protocol: TCP
         - name: metrics
-          containerPort: 9542
+          containerPort: 8100
           protocol: TCP
         livenessProbe:
           httpGet:
             path: /health
```

Comment:
This should now be `/status`, correct, or is there a `/health` endpoint in newer versions? In Enterprise 1.5:

```
# curl -s localhost:8100/health
{"message":"Not found"}

# curl -s localhost:8100/status; echo
{"database":{"reachable":true}, ... "connections_waiting":0}}
```
Applies to the other `/health` checks for readiness and other manifests also. Have not marked them all because they're easy enough to find with grep and the review comments would get rather noisy with each.

Reply:
My bad. I fixed and re-ran the build script.